com.microsoft.mrtk.input/Utilities/PoseSource/PolyfillHandRayPoseSource.cs

```diff
@@ -37,8 +37,8 @@ public override bool TryGetPose(out Pose pose)
             XRNode? handNode = Hand.ToXRNode();
 
             bool poseRetrieved = handNode.HasValue;
-            poseRetrieved &= HandsAggregator != null && HandsAggregator.TryGetJoint(TrackedHandJoint.IndexProximal, handNode.Value, out knuckle);
-            poseRetrieved &= HandsAggregator != null && HandsAggregator.TryGetJoint(TrackedHandJoint.Palm, handNode.Value, out palm);
+            poseRetrieved &= XRSubsystemHelpers.HandsAggregator?.TryGetJoint(TrackedHandJoint.IndexProximal, handNode.Value, out knuckle) ?? false;
```

Comment:
is the ?. safe here? since HandsAggregator is a Unity object and those are not recommended to use null-prop.

if you use the unity plugin for visual studio, it should flag these with warnings

Reply:
In this case, `HandsAggregator` is not a Unity object, so this is safe! At the very base level, `Subsystem`s are just

![image](https://user-images.githubusercontent.com/3580640/212430553-6e5c805e-0dbe-4fa4-af40-85e61ad372a4.png)